gobblin-cluster/src/test/java/org/apache/gobblin/cluster/GobblinHelixJobLauncherTest.java

```diff
@@ -17,6 +17,12 @@
 
 package org.apache.gobblin.cluster;
 
+import com.google.common.base.Optional;
```

Comment:
Please use the Gobblin codestyle
https://gobblin.readthedocs.io/en/latest/developer-guide/CodingStyle/